package.json

```diff
@@ -50,8 +50,8 @@
     "test:ci": "turbo run test --parallel -- --ci --testLocationInResults --json --coverage",
     "test:clear": "pnpm -r exec jest --clearCache",
     "test:coverage": "turbo run test --parallel -- --coverage",
-    "test:e2e": "playwright test --config=packages/app/playwright.config.ts",
-    "test:e2e:contracts": "playwright test --config=packages/e2e-contract-tests/playwright.config.ts",
+    "test:e2e": "pnpm build:crx && playwright test --config=packages/app/playwright.config.ts",
```

Comment:
you can use `run-s` here as we have it setup in the project

Reply:
Fixed

Reply:
That's true, thanks! It's fixed now